packages/beacon-node/src/network/forks.ts

```diff
@@ -54,6 +55,12 @@ export function getActiveForks(config: ChainForkConfig, epoch: Epoch): ForkName[
   return activeForks;
 }
 
+export function getActiveSubscribeBoundaries(config: ChainForkConfig, epoch: Epoch): SubscribeBoundary[] {
+  const activeForks = getActiveForks(config, epoch);
+
+  return activeForks.map((fork) => ({fork}));
```

Comment:
we're not able to delete a `SubscribeBoundary` because we always create new `SubscribeBoundary` objects here
maybe that's reason why sim test is failing?

Reply:
> maybe that's reason why sim test is failing?

seems to be a issue with LH, see https://github.com/ChainSafe/lodestar/pull/7973